.gitignore

```diff
@@ -0,0 +1,8 @@
+Pipfile
+Pipfile.lock
```

Comment:
These both should be checked in, no?

Reply:
added them made pre-commit shout at me, so they were left out